include/envoy/network/proxy_protocol.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "envoy/network/address.h"
+
+namespace Envoy {
+namespace Network {
+
+struct ProxyProtocolHeader {
```

Comment:
Rename `ProxyProtocolOptions`? When I think of `header`, I think of something in wire-format.